gui/slick/interfaces/default/home.mako

```diff
@@ -222,7 +224,7 @@ $(document).ready(function(){
             4: { sorter: 'quality' },
             5: { sorter: 'eps' },
             % if sickbeard.FILTER_ROW:
-            6: { filter : 'parsed' }
+                7: { filter : 'parsed' }
             % endif
         },
```

Comment:
I didn't look into it, but I guess it's the default sort: column 6 ascending or descending (not sure what 1 means), column 2 ascending or descending (the opposite of 1).
So sort by name and active status. Maybe I should change that too, as active is column 7 now?